server/src/test/scala/passengersplits/CanFindASplitForAnApiFlightSpec.scala

```diff
@@ -62,6 +62,7 @@ class CanFindASplitForAnApiFlightSpec extends {
 
       "When we ask for a report of voyage pax splits then we should see pax splits of the 1 passenger in eeaDesk queue" in {
         val flightScheduledDateTime = SDate(2017, 4, 2, 15, 33)
+        Thread.sleep(1000L)
```

Comment:
1) try removing this or 2) reducing it or 3) replacing it with something less brute force